src/foam/nanos/notification/EmailEnabledDAO.java

```diff
@@ -0,0 +1,102 @@
+package foam.nanos.notification;
+import foam.core.Detachable;
+import foam.dao.AbstractSink;
+import foam.nanos.notification.email.EmailMessage;
+import foam.core.FObject;
+import foam.core.X;
+import foam.dao.DAO;
+import foam.dao.ProxyDAO;
+import foam.nanos.app.AppConfig;
+import foam.nanos.auth.User;
+import foam.nanos.logger.Logger;
+import foam.nanos.notification.email.EmailMessage;
+import foam.nanos.notification.email.EmailService;
+
+import java.text.NumberFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import static foam.mlang.MLang.AND;
+import static foam.mlang.MLang.EQ;
+
+public class EmailEnabledDAO extends ProxyDAO {
```

Comment:
Change class name to something more meaningful. EmailEnabledDAO doesn't tell me what this class does.